Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -120,6 +120,40 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
 
           Please make sure to read and understand the documentation page on :ref:`taint effects and unmanaged pods<taint_effects>`.
 
+    .. group-tab:: Kubespray
```

Comment:
I don't think we have too many users of Cilium on Kubespray at the moment? In fact, I've got no idea, but given that we've been using Kind to “get started” and run quick tutorials, I'd assume more readers to be looking for Kind than Kubespray in the docs.

All of this to say, can we move this tab later in the tab list, please? Maybe after Minikube?